ios/Classes/GoogleMapsNavigationViewRegistry.swift

```diff
@@ -14,24 +14,42 @@
 
 class GoogleMapsNavigationViewRegistry {
   private var views: [Int64: GoogleMapsNavigationView] = [:]
+  // Using a concurrent queue with a barrier ensures that write operations are serialized,
+  // meaning each write completes before another write can access the shared resource.
+  // Multiple read operations can still proceed concurrently as long as no write is in progress.
+  private let queue = DispatchQueue(label: "thread_safe_queue", attributes: .concurrent)
```

Comment:
`thread_safe_queue` is not really specific label.
could it be something like
`google_navigation_flutter.thread_safe_view_registry_queue`
or just
`thread_safe_view_registry_queue`

Reply:
Agreed. Let's use `google_navigation_flutter.thread_safe_view_registry_queue`